localMatchApp/log.cpp

```diff
@@ -0,0 +1,31 @@
+#include "log.h"
+
+using namespace std;
+
+log::log()
```

Comment:
Change the name of this module to avoid confusion with debug log

Reply:
changed to logIn